src/ConnectionDispatcher.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <memory>
```

Comment:
@morheit, what is purpose of this header?

Reply:
@malirod , oops, seems I forgot to delete it. `#include <memory>` removed